lib/legacy/legacy-element-mixin.js

```diff
@@ -816,22 +869,19 @@ export const LegacyElementMixin = dedupingMixin((base) => {
      * @param {string} name HTML attribute name
      * @param {boolean=} bool Boolean to force the attribute on or off.
      *    When unspecified, the state of the attribute will be reversed.
-     * @return {boolean} true if the attribute now exists
+     * @param {Element=} node Node to target.  Defaults to `this`.
+     * @return {void}
+     * @override
      */
-    toggleAttribute(name, bool) {
-      let node = /** @type {Element} */ this;
-      if (arguments.length === 3) {
-        node = /** @type {Element} */ arguments[2];
-      }
+    toggleAttribute(name, bool, node) {
+      node = /** @type {Element} */ (node || this);
       if (arguments.length == 1) {
         bool = !node.hasAttribute(name);
       }
       if (bool) {
         node.setAttribute(name, '');
-        return true;
```

Comment:
I think this would break our TS compilation. See recent PRs to tackle this.

Reply:
Ack! Good catch, that was a merge conflict that I misresolved. PTAL